nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -74,6 +74,7 @@ class ONNXDepthwiseConvolutionMetatype(ONNXOpWithWeightsMetatype):
     weight_port_ids = [1]
     bias_port_id = 2
     output_channel_axis = 1
+    input_channel_axis = 1
```

Comment:
You introduced these new attributes, but don't use them. Could you place at least TODO?

Reply:
It is in use. See `node_utils.py`.

Reply:
It is used, but only for OpenVINO

Reply:
Yes, because `SmoothQuant` is implemented only in the `OpenVINO` backend. What are your suggestions?

Reply:
As I can see it is also used in `MinMax` `OpenVINO` backend. 
Should be the approach of detecting the channel axis of activation reused for `MinMax` `ONNX` backend then?

Reply:
If we decide to use the same way for `ONNX` I suggest placing at least `TODO`